apps/src/metrics/StatsigReporter.js

```diff
@@ -135,18 +135,6 @@ class StatsigReporter {
     }
   }
 
-  findOrCreateStableId() {
```

Comment:
Removed, as we are setting the stable_id as a cookie in the `application_controller.rb` as a `before_action` hook. This should always populate the cookie, which will be ready for the client-side code here to use.